src/ui/components/Rating/index.js

```diff
@@ -152,7 +152,7 @@ export class RatingBase extends React.Component<InternalProps, StateType> {
 
       if (readOnly) {
         return (
-          <div {...props}>
+          <div aria-hidden="true" {...props}>
```

Comment:
We could potentially add this into the definition of `props`, above, as we're already using `props` to define this `div`. The change would look like:

```diff
diff --git a/src/ui/components/Rating/index.js b/src/ui/components/Rating/index.js
index d9f469dcd..80a4ccac8 100644
--- a/src/ui/components/Rating/index.js
+++ b/src/ui/components/Rating/index.js
@@ -140,6 +140,7 @@ export class RatingBase extends React.Component<InternalProps, StateType> {
         thisRating - rating > 0.25 && thisRating - rating <= 0.75;
 
       const props = {
+        'aria-hidden': readOnly,
         className: makeClassName('Rating-star', `Rating-rating-${thisRating}`, {
           'Rating-selected-star': isSelected,
           'Rating-half-star': halfStar,
@@ -152,7 +153,7 @@ export class RatingBase extends React.Component<InternalProps, StateType> {
 
       if (readOnly) {
         return (
-          <div aria-hidden="true" {...props}>
+          <div {...props}>
             {this.renderStar({
               half: halfStar,
               selected: isSelected,
```